code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -30,7 +30,7 @@
 	new /obj/item/clothing/gloves/color/captain(src)
 	new /obj/item/restraints/handcuffs/cable/zipties(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/gun/ballistic/revolver/golden
```

Comment:
Ten rewolwer używa amunicji .357, której IIRC nie da się dostać w żaden sposób na stacji, a jeś;li się nie mylę, to da się go zrekonstruować aby dostać illegal tech.